src/Accelerators/NNPA/zdnn.cmake

```diff
@@ -38,7 +38,7 @@ function(setup_zdnn version)
       BUILD_COMMAND sh -c "export MAKEFLAGS=--no-print-directory && \
                          make -q -C zdnn lib/libzdnn.so && true || \
                          (MAKEFLAGS=--no-print-directory \
-                          make -j$(nproc) -C zdnn lib/libzdnn.so && \
+                          make -C zdnn lib/libzdnn.so && \
```

Comment:
I think we all have used this when building onnx-mlir for NNPA, and our jenkins uses this also and haven't seen this error. I wonder under which circumstance you got this, perhaps, the local environment is different.

Can you confirm if your machine has `nproc` program? `$(nproc)` is to invoke `nproc`.